src/main/java/net/dv8tion/jda/core/audio/AudioWebSocket.java

```diff
@@ -447,7 +447,7 @@ private InetSocketAddress handleUdpDiscovery(InetSocketAddress address, int ssrc
             // the last 2 bytes. Not sure why these exist.  The last 2 bytes are the port. More info below.
             String ourIP = new String(receivedPacket.getData());//Puts the entire byte array in. nulls are converted to spaces.
             ourIP = ourIP.substring(0, ourIP.length() - 2); //Removes the port that is stuck on the end of this string. (last 2 bytes are the port)
-            ourIP = ourIP.trim();                           //Removes the extra whitespace(nulls) attached to both sides of the IP
+            ourIP = ourIP.substring(4, ourIP.length()).trim();  //Removes the SSRC of the answer package and the extra whitespace(nulls) attached to both sides of the IP
```

Comment:
This could be in the line above as `substring(4, ourIP.length() - 2)` to only keep the trim on that line